src/Ast/PhpDoc.php

```diff
@@ -415,7 +415,7 @@ protected function toLines(Map $info): Vector
                 $code = Formatter::format("<?php\n{$code}");
                 $code = str_replace('[ELLIPSIS]', '[...]', $code);
                 $code = Vector::new(explode("\n", $code))
-                    ->skip(2)->skipLast(1)
+                    ->skip(1)->skipLast(1)
```

Comment:
Nit: could you add a comment for this line that indicates what it is doing? I'm not sure what affect this change has.

Reply:
Good call, done!